src/common/veteran.js

```diff
@@ -259,7 +259,7 @@ const completeVeteran = {
     value: '1235551234',
     dirty: false
   },
-  understandsFinancialDisclosure: true,
+  understandsFinancialDisclosure: makeField(true),
```

Comment:
`makeField` sets the dirty flag to false by default. i kept the value as true because that is what it was set as before, is there a reason to set it to false? this object represents a completed veteran form so i think `value: true` makes sense here, with `value: false` for the blank veteran object as set on L56